src/gui/plugins/scene3d/Scene3D.cc

```diff
@@ -441,6 +441,9 @@ void IgnRenderer::Render()
   if (!this->dataPtr->followTarget.empty())
   {
     rendering::ScenePtr scene = this->dataPtr->renderUtil.Scene();
+    if (!scene)
```

Comment:
How about getting the scene once at the beginning of `IgnRenderer::Render()` and printing a warning? This way, if we add other scene calls to the body of this function, we don't need to remember to check for `nullptr`.

Reply:
Fixed here 3008548703b50ab7916bcba68e047c4bd5c456c8 .  I might need some help on the warning message, wasn't sure what exactly to put

Reply:
That looks good to me